test/Microsoft.ApplicationInsights.AspNetCore.Tests/TelemetryInitializers/ComponentVersionTelemetryInitializerTests.cs

```diff
@@ -48,19 +47,17 @@ public void InitializeDoesNotOverrideExistingVersion()
         [Fact]
         public void InitializeDoesNotThrowIfVersionDoesNotExist()
         {
-            var config = new ConfigurationBuilder()
-                .Build();
-            var initializer = new ComponentVersionTelemetryInitializer(config);
+            var initializer = new ComponentVersionTelemetryInitializer(this.BuildConfigurationWithVersion(null));
             var telemetry = new RequestTelemetry();
             initializer.Initialize(telemetry);
         }
 
-        private IConfigurationRoot BuildConfigurationWithVersion()
+        private IOptions<ApplicationInsightsServiceOptions> BuildConfigurationWithVersion(string versions = "1.0.0")
         {
-            Environment.SetEnvironmentVariable(VersionKey, "1.0.0");
```

Comment:
@SergeyKanzhelev Do you know what the significance of this environment variable setting was?